_source/_posts/2024-02-29-third-party-cookies.md

```diff
@@ -0,0 +1,70 @@
+---
+layout: blog_post
+title: "The End of Third-Party Cookies"
+author: edunham
+by: advocate
+communities: [devops,security,mobile,.net,java,javascript,go,php,python,ruby]
+description: "Google Chrome is deprecating all third-party cookies in 2024. How will this affect your Okta application, remember me features, and Okta Sign-In Widget?"
+tags: []
+tweets:
+- ""
+- ""
+- ""
+image: blog/3pc/social.jpg
+type: awareness
+---
+
+## What are third-party cookies?
+
+Cookies are as old as the internet. Historically, cookies were among the only options for personalizing a user's online experience and carrying their preferences from page to page. First-party cookies are issued by the web site where they're used, and third-party cookies come from other domains. 
+
+Third-party cookies allow user behavior to be tracked across different sites. These cookies are now widely abused to collect and share users' data. For the legitimate use cases which used to require third-party cookies, like federated logins and multi-brand identity providers, more secure options are actively being developed. 
+
+Today, the drawbacks to users' security and privacy from third-party cookie implementations outweigh their benefits so much that all major browsers are phasing them out. Safari has blocked third-party cookies for years, and Firefox retricts third-party cookies associated with trackers. Chrome is now [phasing out third-party cookies](https://developers.google.com/privacy-sandbox/3pcd) in 2024. 
+
+If a user has a cookie from okta.com in their browser, that cookie will count as first-party when accessed by the okta.com website, and it will count as third-party when accessed from a website on any other domain.
+
+When a user logs into their Okta account in a web browser, a [session cookie](https://developer.okta.com/docs/guides/session-cookie/main/#about-okta-session-cookies) stores state information about their login session. These cookies are usually first-party, but in some situations they can be third-party. If your code uses Okta features that rely on third-party cookies, this means that you'll need to make some changes to keep the identity experience working as intended. 
```

Comment:
> rely on third-party cookies

I'm not sure if this distinction will make things more clear or not, but features at Okta may depends on cookies, which may be 1st or 3rd party. Whether or not those cookies are 1p or 3p depends on the domains